config.json

```diff
@@ -0,0 +1,153 @@
+﻿{
+  "settings": {
+    "OSGroup": {
+      "description": "Target OS for result binaries.",
```

Comment:
"target OS" and "target os group" are not the same.  I'd be explicit and add "group" to the description. For that matter,  it might be nice to link to a description of the osgroup property.  I'm not certain if we have an MD for for this or not though.